src/parquet/column/page.h

```diff
@@ -169,14 +170,16 @@ class PageWriter {
  public:
   virtual ~PageWriter() {}
 
-  virtual void Close() = 0;
+  // TODO think of a better way to pass column chunk statistics
+  virtual void Close(const std::shared_ptr<RowGroupStatistics>& chunk_statistics) = 0;
```

Comment:
The API is admittedly rather ugly here, because PageWriter operates on raw bytes and not values. It could handle _column chunk_ statistics by itself, but then it would have to be initialized somewhere with a concrete type.